kb/plane_geometry/formal_statements/statements/statements_scs/stat_regular_hexagon_internal_angle_value_degree.scs

```diff
@@ -0,0 +1,14 @@
+stat_regular_hexagon_internal_angle_value_degree <- statement;;
+
+stat_regular_hexagon_internal_angle_value_degree -> rrel_key_sc_element: concept_regular_hexagon; nrel_internal_angle; nrel_value; rrel_degree;;
+
+stat_regular_hexagon_internal_angle_value_degree => nrel_main_idtf: [Акс.(правильный шестиугольник;внутренний угол*;значение*;угловой градус')] (*<- lang_ru;;*);;
+stat_regular_hexagon_internal_angle_value_degree => nrel_main_idtf: [Ax.(regular hexagon;internal angle*;value*;degree')] (*<- lang_en;;*);;

```

Comment:
It isn't an axiom. It is a statement. You should use [Stat.(...)] insted of [Ax.(...)]